src/test/java/Homework24.java

```diff
@@ -0,0 +1,46 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+
+    public class Homework24 extends BaseTest {
+
+        @Test
+        public void deletePlaylist() throws InterruptedException {
+
+            String expecteddeletedPlaylistmsg = "Deleted playlist \"test pro playlist.\"";
+
+            provideEmail("aparajita.jha@testpro.io");
+            providePassword("testpro135@");
+            clickSubmit();
+            Thread.sleep(2000);
+            openPlaylist();
+            clickDeletePlaylistBtn();
+
+
+            Assert.assertEquals(getPlaylistDeletedmsg(), expecteddeletedPlaylistmsg);
```

Comment:
Please, delete `throws InterruptedException`

**Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.**

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.